plugin/xcodebuild.vim

```diff
@@ -72,6 +73,11 @@ function!s:scheme_name()
   return s:chosen_scheme
 endfunction
 
+function! s:set_scheme(scheme)
```

Comment:
Should we move this up with `build` and `test`? I feel like it makes sense to group actions together, but maybe I'm insane?